framework/console/controllers/BaseMigrateController.php

```diff
@@ -663,6 +677,13 @@ public function actionCreate($name)
                 return ExitCode::IOERR;
             }
 
+            if ($this->newFileMode !== null) {
+                @chmod($file, $this->newFileMode);
```

Comment:
I think we should catch the failures here and with ownership below and display the information because when user sets these it's rather important to know whether it failed or not.

Reply:
`chmod` is used throughout Yii with the @ infront of it, so I used it too. But I agree that it's better without it.

Reply:
@samdark What are your thoughts on this?

Reply:
What would be Windows behavior in this case?

Reply:
I don't mind `@` - it's the `false` I would alert.

Reply:
@samdark On windows `print_r(chmod('c:\tmp\test.txt',0770));` returns `1`.
So if the result is `false` we could catch it.
Should we log an error or throw an exception?

Reply:
Raising an exception sounds better.

Reply:
Trying to change the ownership to a non existing user now raises:
Error: Unable to change file ownership of "/app/console/migrations/m210523_185610_test2.php" to "test:test"